### PR TITLE
Add 3.0.8 release note

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:
SR v3.0.8 is released.
What I'm doing:
To add 3.0.8 release note.
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/release_notes/release-3.0.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ displayed_sidebar: "English"
...
+
+## Bug Fixes
+
+Fixed the following issues:
+
+- When `show proc '/current_queries';` is being executed and meanwhile a query begins to be executed, BEs may crash. [#34316](https://github.com/StarRocks/starrocks/pull/34316)
+- When data is continuously loaded into a Primary Key table with a sort key specified at a high frequency, compaction failures may occur. [#26486](https://github.com/StarRocks/starrocks/pull/26486)
+- If a filtering condition is specified in a Broker Load job, BEs may crash during the data loading in certain circumstances. [#29832](https://github.com/StarRocks/starrocks/pull/29832)
+- An unknown error is reported when SHOW GRANTS is executed. [#30100](https://github.com/StarRocks/starrocks/pull/30100)
+- BE may crash for specific data types if the target data type specified in the `cast()`  function is the same as the original data type. [#31465](https://github.com/StarRocks/starrocks/pull/31465)
```
Comment (amber-create) [inline docs/en/release_notes/release-3.0.md:23]: image
extra space

File: docs/en/release_notes/release-3.0.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ displayed_sidebar: "English"
...
+- `DATA_TYPE` and `COLUMN_TYPE` for BINARY or VARBINARY data types are displayed as `unknown` in the `information_schema.columns` view. [#32678](https://github.com/StarRocks/starrocks/pull/32678)
+- Long-time, frequent data loading into a Primary Key table with persistent index enabled may cause BEs to crash. [#33220](https://github.com/StarRocks/starrocks/pull/33220)
+- The query result is incorrect when Query Cache is enabled. [#32778](https://github.com/StarRocks/starrocks/pull/32778)
+- After a cluster is restarted, the data in a restored table may be inconsistent with the data in that table before being backed up. [#33567](https://github.com/StarRocks/starrocks/pull/33567)
+- If RESTORE is executed and meanwhile Compaction takes place, it may cause BEs crash. [#32902](https://github.com/StarRocks/starrocks/pull/32902)
```
Comment (amber-create) [inline docs/en/release_notes/release-3.0.md:28]: image
grammatical error? -> cause BEs to crash/cause a BE crash

**Review by amber-create — DISMISSED** [review]

File: docs/zh/release_notes/release-3.0.md [hunk 3]
```diff
@@ -4,6 +4,30 @@ displayed_sidebar: "Chinese"
...
+- 执行 `show proc '/current_queries';` 时，如果某个查询刚开始执行，可能会引起 BE Crash。[#34316](https://github.com/StarRocks/starrocks/pull/34316)
+- 带 Sort Key 的主键模型表持续高频导入时，会小概率出现 Compaction 失败。[#26486](https://github.com/StarRocks/starrocks/pull/26486)
+- 修改 LDAP 认证用户的密码后，使用新密码登录时报错。
+- 如果提交的 Broker Load 作业包含过滤条件，在数据导入过程中，某些情况下会出现 BE crash。[#29832](https://github.com/StarRocks/starrocks/pull/29832)
+- SHOW GRANTS 时报 `unknown error`。[#30100](https://github.com/StarRocks/starrocks/pull/30100)
+- 当使用函数 `cast()` 时数据类型转换前后类型一致，某些类型下会导致 BE crash。[#31465](https://github.com/StarRocks/starrocks/pull/31465)
+- BINARY 或 VARBINARY 类型在 `information_schema.columns` 视图里面的 `DATA_TYPE` 和 `COLUMN_TYPE` 显示为 `unknown`。[#32678](https://github.com/StarRocks/starrocks/pull/32678)
+- 长时间向持久化索引打开的主键模型表高频导入，可能会引起 BE crash。[#33220](https://github.com/StarRocks/starrocks/pull/33220)
+- Query Cache 开启后查询结果有错。[#32778](https://github.com/StarRocks/starrocks/pull/32778)
+- 重启后 RESTORE 的表中的数据和 BACKUP 前的表中的数据在某些情况话会出现不一致的情况。[#33567](https://github.com/StarRocks/starrocks/pull/33567)
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.0.md:28]: 情况话 --- 情况下

File: docs/zh/release_notes/release-3.0.md [hunk 4]
```diff
@@ -4,6 +4,30 @@ displayed_sidebar: "Chinese"
 
 # StarRocks version 3.0
 
+## 3.0.8
+
+发布日期：2023 年 11 月 20 日
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.0.md:9]: 这个日期是不是要写 周五 11 月 17 日

File: docs/en/release_notes/release-3.0.md [hunk 5]
```diff
@@ -4,6 +4,29 @@ displayed_sidebar: "English"
 
 # StarRocks version 3.0
 
+## 3.0.8
+
+Release date: November 20, 2023
+
+## Improvements
+
+- The `COLUMNS` table in the system database `INFORMATION_SCHEMA` can display ARRAY, MAP, and STRUCT columns. [#33431](https://github.com/StarRocks/starrocks/pull/33431)
```
Comment (evelynzhaojie) [inline docs/en/release_notes/release-3.0.md:13]: table -- view

File: docs/en/release_notes/release-3.0.md [hunk 6]
```diff
@@ -4,6 +4,29 @@ displayed_sidebar: "English"
 
 # StarRocks version 3.0
 
+## 3.0.8
+
+Release date: November 20, 2023
```
Comment (evelynzhaojie) [inline docs/en/release_notes/release-3.0.md:9]: November 17?

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**